src/index.ts

```diff
@@ -220,20 +220,22 @@ class ServerlessSSMPublish {
         Name: param.path,
         Description: param.description || `Placed by ${this.serverless.service.getServiceName()} - serverless-ssm-plugin`,
         Value: typeof param.value === 'string' ? param.value : yaml.safeDump(param.value),
-        Overwrite: true,
-        Type: param.secure ? 'SecureString' : 'String',
+        Type: param.type ? param.type :  param.secure ? 'SecureString' : 'String',
```

Comment:
```suggestion
       Overwrite: true,
       Type: param.type ? param.type : param.secure ? 'SecureString' : 'String',
```

We will need the overwrite flag to stay set, otherwise the plugin won't update existing parameters if they change.

Reply:
Thanks for highlighting. I will fix this.